### PR TITLE
[Old Blue] Add global, index.build rules

### PR DESCRIPTION
**Pull request by AprilSylph**
- diffed index.build.css
- diffed global.css
- bumped version to 1.0.0 since this is a very whole and stable version now
- added a note in the extension's preferences with a link to the raw source CSS file for easy usage in userstyle addons (im now using it in Stylus and its very nice)

### CODE REVIEW

**Comment by AprilSylph** [comment]
i thought i had done global.css already but in my initial panic what i _actually_ did was grab psi's gist and reverse the rules there, and then assumed he had only grabbed global.css when it turned out to only be one of the files. it was actually global.build.css :')

File: Extensions/old_blue.css [hunk 1]
```diff
@@ -12,1047 +12,225 @@
...
-.yam-plus-ad-container .ad-bg.bg-blue {
-	background-color: var(--old-accent);
+.ui_peepr_glass {
+	background-color: rgba(54,70,93,0.95);
 }
 
-.yam-plus-ad-container .ad-bg.bg-grey {
-	background-color: #687486;
+.ui_peepr_glass.opaque {
+	background-color: var(--old-blue);
```
Comment (nightpool) [inline Extensions/old_blue.css]: what's the diff between this and `#36465d`?

File: Extensions/old_blue.css [hunk 2]
```diff
@@ -12,1047 +12,225 @@
...
 }
 
-.yam-plus-ad-container .ad-bg.bg-red {
-	background-color: var(--old-red);
+.indash_header_wrapper .editable_text {
+	border-color: var(--old-accent);
 }
 
-.ui_peepr_glass {
-	background-color: rgba(0,25,53,0.95);
```
Comment (nightpool) [inline Extensions/old_blue.css]: I certainly fucked that one up, huh

File: Extensions/old_blue.js [hunk 3]
```diff
@@ -9,6 +10,13 @@ XKit.extensions.old_blue = new Object({
 
 	running: false,
 
+	preferences: {
+		"note": {
+			text: `<span style="text-transform:none"><a href="https://raw.githubusercontent.com/new-xkit/XKit/master/Extensions/old_blue.css" target="_blank">CSS file</a> for usage in userstyle add-ons</span>`,
```
Comment (nightpool) [inline Extensions/old_blue.js]: not sure how I feel about the wording here, could stand to be a little more comprehensible

File: Extensions/old_blue.css [hunk 4]
```diff
@@ -12,1047 +12,225 @@
...
 
-.media-container.audio-media-container .album-art-uploader .media-dropzone {
-	color: var(--old-purple);
+.tab_post_type .post_type_icon {
+	background-image: url("https://new-xkit.github.io/XKit/Extensions/dist/quick_compose_sprite.png");
 }
 
-.media-dropzone .media-url {
-	color: var(--old-text);
+@media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (-moz-min-device-pixel-ratio: 2), only screen and (min-device-pixel-ratio: 2), only screen and (min-resolution: 192dpi), only screen and (min-resolution: 2dppx) {
```
Comment (nightpool) [inline Extensions/old_blue.css]: don't we already got one of these? or is this a different media query or smth?

Comment (AprilSylph) [reply Extensions/old_blue.css]: different media query

File: Extensions/old_blue.js (hunk 3 above)
Comment (AprilSylph) [reply Extensions/old_blue.js]: open to suggestions, will give it another go myself tomorrow

**Pull request merged**